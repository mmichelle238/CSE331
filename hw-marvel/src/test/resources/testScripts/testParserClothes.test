# Test the parser on a graph that contains several nodes
# with multiple edges.

LoadGraph clothes newClothes.tsv

ListChildren clothes pants

ListChildren clothes shirts

ListChildren clothes overalls