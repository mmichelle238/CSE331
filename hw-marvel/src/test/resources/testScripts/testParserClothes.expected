# Test the parser on a graph that contains several nodes
# with multiple edges.

loaded graph clothes

the children of pants in clothes are: jeans(pantbook) overalls(pantbook) shorts(pantbook)

the children of shirts in clothes are: blouses(shirtbook) overalls(shirtbook) sweaters(shirtbook) sweatshirts(shirtbook)

the children of overalls in clothes are: blouses(shirtbook) jeans(pantbook) pants(pantbook) shirts(shirtbook) shorts(pantbook) sweaters(shirtbook) sweatshirts(shirtbook)